docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -1408,10 +1404,46 @@ Wave III
     - Spec. mods
     - Note
   * - 14.0
+    - Update to GBD 2021 data 
+    - Baseline
+    - Baseline
+    - 
+    - 
```

Comment:
We should note that we'll want to have this be run on results from the pregnancy sim that have been updated to 2021... so we should probably plan to do the pregnancy sim update before the child sim since the child sim uses the pregnancy sim outputs